envi/tests/test_arch_arm.py

```diff
@@ -19,10 +19,9 @@
 
 
 ''' 
-
   This dictionary will contain all commands supported by ARM to test
   Fields will contain following information:
-  archBitMask, ophex, va, repr, flags, emutests
+  archBitMask, OPHEX, VA, repr, flags, emutests
```

Comment:
no thank you.  please keep the CAPITALS to a minimum.  just an aesthetic, but something i feel strongly about.